Federal Data/753 Local Unit Population and HouseHold/Pradesh 6/Dailekh/Bhagawatimai Gaunpalika.csv

```diff
@@ -1,12 +1,9 @@
-,,, ,
-,,,,
-Ward No.,Household ,Total Population ,Total Female ,Total Female 
-,,,,
+Ward No.,Household,Total Population,Total Female,Total Female

```

Comment:
`Female` twice in the headers. Should be Male, then Female, I believe.

Reply:
Yes, that should be Male then Female. I missed it while looking.